blueprint/api.apib

```diff
@@ -2174,6 +2315,30 @@ This endpoint allows you to check whether a username is valid (by running a vali
 + data(Slugged Route Organization Resource)
 + include JSON API Version
 
+## Stripe Plan Attributes (object)
++ amount: 999 (number)
++ created: `2016-07-08T03:03:51.967Z` (string)
++ name: `Emerald Personal` (string)
+
+##Stripe Plan Resource Identifier
++ id: `1` (string, required)
++ type: `stripe-plan` (string, required)
+
+##Stripe Plan Resource (object)
++ include Stripe Plan Resource Identifier
++ attributes(Stripe Plan Attributes)
++ relationships
+    + project (object)
+        + data(Project Resource Identifier)
+
+## Stripe Plan Response
++ data(Stripe Plan Resource)
++ include JSON API Version
+
+## Stripe Plans Response
++ data(array[Stripe Plan Resource], required)
```

Comment:
I think I added it because I was looking at the Tasks Response below.  I noticed now that required is on the "Tasks Response" as well as the "Comments Response". Should those be removed as well?